src/components/Button/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import * as SC from "./Button.styles";
+import {
+  setButtonStyles,
+  ButtonSize,
+  ButtonVariant,
+  ButtonColor,
+} from "./Button.utils";
+
+export interface ButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  size?: ButtonSize;
+  variant?: ButtonVariant;
+  color?: ButtonColor;
+  children: React.ReactNode;
+}
+
+const Button: React.FC<ButtonProps> = ({
+  size = "md",
+  variant = "contained",
+  color = "serenity",
+  children,
+  ...props
+}) => {
```

Comment:
```suggestion
export interface ButtonProps
  extends ButtonHTMLAttributes<HTMLButtonElement> {
  size?: ButtonSize;
  variant?: ButtonVariant;
  color?: ButtonColor;
}

const Button = ({
  size = "md",
  variant = "contained",
  color = "serenity",
  children,
  ...props
}: PropsWithChildren<ButtonProps>) => {
```

P2:

1. 컴포넌트의 타입을 React.FC를 활용하시는 이유가 있을까요?
https://emewjin.github.io/why-not-fc/

2. children 타입을 선언하는 경우, PropsWithChildren을 활용해봐요.
`children: React.ReactNode`: React에서 지원하는 타입을 활용하면 더 명시적으로 선언할 수 있어요.

3. React 모듈을 직접 쓰기보다 직접 쓰기보단, 필요한 모듈을 import 해봐요.
`React.ButtonHTMLAttributes` -> `ButtonHTMLAttributes`
tree-shaking에 대해 학습해봐도 좋겠어요.